contrib/federations_tutorial/node0/elements.conf

```diff
@@ -0,0 +1,13 @@
+rpcuser=user1
+rpcpassword=password1
+rpcport=18892
+port=18893
+
+connect=localhost:18889
+connect=localhost:18891
+
+initialfreecoins=50000000000000
+regtest=0
```

Comment:
You don't need this. Also instead of regtest=0 or regtest=1 you can just use chain=mycustomname and it should work fine

Reply:
Which part is not needed?

Reply:
Which part is not needed?

Reply:
regtest=0, I think it's clearer to use the chain parameter instead, see https://github.com/ElementsProject/elements/blob/elements-0.14.1/src/chainparamsbase.cpp#L18

Reply:
Ah right!